src/SerializeAST.h

```diff
@@ -9,6 +9,7 @@
 #ifndef SNOWCRASH_SERIALIZE_AST_H
 #define SNOWCRASH_SERIALIZE_AST_H
 
+#include "StringUtility.h"
```

Comment:
Since it is not needed for anything in the header it would suffice to include it in the .cc file no?